### ISSUE TITLE
The Machine Learning (Computer Vision) Pipeline

### ISSUE THREAD

**Issue by nickvazz** [truc-h-nguyen/Toddler-activity-suggestions#4]
1) Get Data we want to use
   a) preprocess
   b) **[maybe if we use multiple]** combining our datasets
   c) **[maybe]** collect our own data (separate project -- maybe)
2) Get Neural Network Models we want to use
   a) transfer learning ([keras tutorial](https://keras.io/guides/transfer_learning/))
   b) **[maybe]** make our own more complicated model on top of the transfer learning model
   c) **[maybe but probably not]** re-use a different state of the art model (SOTA) -- go find some interesting models on GitHub to try and use
3) Training
   a) basic training loop
   b) hyper parameters (learning rate, optimization function, number of layers, how many convolutions)
4) Evaluate our model
